packages/webapp/src/containers/Filter/types.ts

```diff
@@ -20,6 +20,7 @@ export interface FilterState {
   [filterOptionId: string]: /* e.g. ABANDONED, COMPLETED */ {
     active?: boolean;
     label?: string /* e.g. 'Abandoned', 'Completed */;
+    [key: string]: any;
```

Comment:
Ah it looks like this type is mostly used for multi-select right now. We might want to make the difference more explicit in the future!

Reply:
I can "defend" myself with TypeScript but I'm not any kind of TS wizard, so I start having troubles if typing things become a bit complex. I just wanted to get rid of the error 😆  But I agree that it should be more specific.